pkg/upgrade/upgrade.go

```diff
@@ -400,6 +411,22 @@ func removeCsv(c client.Client, r *rest.Config) error {
 	return nil
 }
 
+func RemoveUICR(cfg *rest.Config, reousrce schema.GroupVersionResource, instanceName string, applicationNS string) error {
```

Comment:
I wonder if we need to export this `func`?

Reply:
And there's a typo `reousrce`

Reply:
rename to `removOdhAppsCR` better?

Reply:
or `removOdhApplicationsCR()`

Reply:
I think `removeResource` is what this `func` does